CMakeLists.txt

```diff
@@ -1,73 +1,72 @@
-cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR) # Policies <= CMP0097 default to NEW
 
-message(AUTHOR_WARNING "If the build fails, please try the autotools/qmake method.")
-
-if (POLICY CMP0074)
-    cmake_policy(SET CMP0074 NEW)
-endif()
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
 
 # TODO: fix the macOS bundle stuff
-if (POLICY CMP0080)
-    cmake_policy(SET CMP0080 OLD)
+cmake_policy(SET CMP0080 OLD)
+
+# CMAKE_MSVC_RUNTIME_LIBRARY must be defined before project() or enable_language() to take effect
+if (MSVC)
+        set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<BOOL:${MSVC_RUNTIME_DYNAMIC}>:DLL>")
 endif()
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
```

Comment:
This was the correct way to go.

Reply:
@zeule 
It's currently `set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)` right at the top of the file.
you expect qBittorrent to need more than one module directory?

Reply:
https://github.com/qbittorrent/qBittorrent/pull/12746#discussion_r449763564